src/spyglass/lfp/lfp_merge.py

```diff
@@ -7,44 +7,24 @@
 from spyglass.common.common_nwbfile import AnalysisNwbfile
 from spyglass.lfp.v1.lfp import LFPV1, ImportedLFPV1
 from spyglass.utils.dj_helper_fn import fetch_nwb
+from spyglass.utils.dj_merge_tables import Merge
 
 schema = dj.schema("lfp_merge")
 
 
 @schema
-class LFPOutput(dj.Manual):
+class LFPOutput(Merge):
     definition = """
-    lfp_id: uuid
-    ---
-    source: varchar(40)
-    version: int
-
+    merge_id: uuid
     """
 
     class LFPV1(dj.Part):
         definition = """
         -> master
```

Comment:
Is there a downside to referencing `LFPOutput` here to be consistent instead of `master`?

```suggestion
        -> LFPOutput
```

Reply:
They're synonymous in this context. I have a slight preference for `-> master` because I do a lot of folder-wide searches for table names and looking at table `definition` properties, and I find it easier to narrow in on what I'm looking for with `-> master` in these cases. It there a reason to prefer explicit?